packages/web-console/src/js/console/grid.js

```diff
@@ -603,12 +572,12 @@ export function grid(root, msgBus) {
   }
 
   function removeClass(e, className) {
-    e.className = e.className.replace(className, '')
+    e.className = e.className.replace(className, '').trim()
   }
 
   function addClass(e, className) {
     if (e && !e.className.includes(className)) {
-      e.className += className
+      e.className += ' ' + className
```

Comment:
no need to change anything, just want to share info about `classlist`, might be helpful for lines like this.

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
hmm, do that specifically did not work for me, let me retry

Reply:
ah, `e.classList.add()` - thanks!

Reply:
addressed in https://github.com/questdb/ui/pull/86/commits/9c85caacdeda3c70a6465cdb5caab1b9df8f4565